packager/media/base/aes_encryptor.cc

```diff
@@ -129,17 +129,40 @@ bool AesCbcEncryptor::CryptInternal(const uint8_t* plaintext,
                                     size_t plaintext_size,
                                     uint8_t* ciphertext,
                                     size_t* ciphertext_size) {
-  const size_t residual_block_size = plaintext_size % AES_BLOCK_SIZE;
+
   const size_t num_padding_bytes = NumPaddingBytes(plaintext_size);
-  // mbedtls requires a buffer large enough for one extra block.
```

Comment:
I think this was still useful context on the calculation of required_ciphertext_size.  Please restore this!